docs/advanced-topics/kubernetes-dashboard.md

```diff
@@ -0,0 +1,134 @@
+# Kubernetes Dashboard Access and Authentication
+
+## Default Setup
+
+    kubernetesDashboard:
+      adminPrivileges: true
+      insecureLogin: false
+
+In the default setup the Dashboard is configured using the `--auto-generate-certificates` flag and has Admin privileges
+
+###### Access the dashboard  using `kubectl proxy` command:
+
+    http://localhost:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/
+
+
+## Expose the Dashboard using a ELB with self-signed certificates
+
+kubernetesDashboard:
+  adminPrivileges: false
+  insecureLogin: false
+
+Ex.
+
+    apiVersion: v1
+    kind: Service
+    metadata:
+      annotations:
+        external-dns.alpha.kubernetes.io/hostname: "kubedash.example.com"
+      name: kubernetes-dashboard
+      labels:
+        run: kubernetes-dashboard
+      namespace: kube-system
+    spec:
+      type: LoadBalancer
```

Comment:
Should we add this as a config option? I guess I still don't understand why the dashboard can no longer be access via the kubeapi ELB which already exists, that's already SSL end to end and uses the certs we generate.

Reply:
Are you using `basic-auth` to access the kubernetes API? How do you authenticate against the API in browser when using a client certificate?

Reply:
Just the original kube-aws mechanism using client SSL certs. i.e. add the CA to the approved ones and the admin key to the relevant keychain for client SSL certs. Then when browsing to the dashboard via kubeapi ELB one receives a client SSL cert prompt to select which admin key.

Reply:
Ok, thanks for the info. Never used it like this. The good news is that I tested and it works fine with the default configuration and it's something like: `https://kubeapi.example.com/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy`

Reply:
@camilb great. I'd like to try to enable token auth via this mechanism without needing to dish out the client SSL cert (as that allows super admin access).